src/storage/Batch.ts

```diff
@@ -111,38 +111,38 @@ export class Batch extends EventEmitter {
         }, this._closeTimeout * this.retries)
     }
 
-    done() {
+    done(): void {
         this.doneCbs.forEach((doneCb) => doneCb())
         this.doneCbs = []
     }
 
-    clear() {
+    clear(): void {
         this.logger.debug('cleared')
         clearTimeout(this._timeout)
         this.streamMessages = []
         this._setState(Batch.states.INSERTED)
     }
 
-    push(streamMessage: Protocol.StreamMessage, doneCb?: DoneCallback) {
+    push(streamMessage: Protocol.StreamMessage, doneCb?: DoneCallback): void {
         this.streamMessages.push(streamMessage)
-        this.size += Buffer.from(streamMessage.serialize()).length
+        this.size += Buffer.byteLength(streamMessage.serialize())
```

Comment:
See commit message in 8141e87 for elaboration.

Reply:
N-I-C-E :+1: